doc/releases/changelog-dev.md

```diff
@@ -243,6 +243,10 @@
 <h3>Bug fixes 🐛</h3>
 
 * `jax.jit` now works with `qml.sample` with a multi-wire observable.
+  [(#5422)](https://github.com/PennyLaneAI/pennylane/pull/5422)
+
+* `qml.qinfo.quantum_fisher` now works with non-`default.qubit` devices.
+  [(#5423)](https://github.com/PennyLaneAI/pennylane/pull/5423)
```

Comment:
Seems like a merge artifact. Likely there's a duplicate entry somewhere else in the changelog.

Reply:
seems to have gone away when I updated with master.